web/angular-wallet/src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
```

Comment:
I create a shareable module.... we should move other utilities herein... like convertNQTPipe etc.

Reply:
There's already a shared module `FuseSharedModule`

Reply:
Thought it is for Fuse only...do you recommend to use it?

Reply:
Yeah I've been putting stuff in there already... might as well not have two shared modules if we can avoid it.

Reply:
We could avoid it.... but it does not feel right. Two arguments against this:
- Fuse is generic and not app specific, i.e. semantically it's not "pure" to reuse it
- `app` and `@fuse` folder are on same level...`burstAmountPipe` needs i18nService, which is part of `app`, so it would reference `app` in `fuse` context. This makes it feel clunky